src/samples/GroupChannelCategorizeByCustomType.js

```diff
@@ -0,0 +1,633 @@
+
+                    updateState({ ...state, messages: updatedMessages, messageInputValue: "" });
+
+                })
+                .onFailed((error) => {
+                    console.log(error)
+                    console.log("failed")
+                });
+        }
+    }
+
+    const onFileInputChange = async (e) => {
+        if (e.currentTarget.files && e.currentTarget.files.length > 0) {
+            const { currentlyJoinedChannel, messages } = state;
+            const fileMessageParams = new FileMessageCreateParams();
+            fileMessageParams.file = e.currentTarget.files[0];
+            currentlyJoinedChannel.sendFileMessage(fileMessageParams)
+                .onSucceeded((message) => {
+                    const updatedMessages = [...messages, message];
+                    updateState({ ...state, messages: updatedMessages, messageInputValue: "", file: null });
+
+                })
+                .onFailed((error) => {
+                    console.log(error)
+                    console.log("failed")
+                });
+
+        }
+    }
+
+    const handleDeleteMessage = async (messageToDelete) => {
+      const { currentlyJoinedChannel } = state;
+      await deleteMessage(currentlyJoinedChannel, messageToDelete); // Delete
+    }
+
+    const updateMessage = async (message) => {
+        updateState({ ...state, messageToUpdate: message, messageInputValue: message.message });
+    }
+
+    const handleLoadMemberSelectionList = async () => {
+        updateState({ ...state, currentlyJoinedChannel: null });
+        const [users, error] = await getAllApplicationUsers();
+        if (error) {
+            return onError(error);
+        }
+        updateState({ ...state, currentlyJoinedChannel: null, applicationUsers: users, groupChannelMembers: [sb.currentUser.userId] });
+    }
+
+    const addToChannelMembersList = (userId) => {
+        const groupChannelMembers = [...state.groupChannelMembers, userId];
+        updateState({ ...state, groupChannelMembers: groupChannelMembers });
+
+    }
+
+    const setupUser = async () => {
+        const { userNameInputValue, userIdInputValue } = state;
+        const sendbirdChat = await SendbirdChat.init({
+            appId: SENDBIRD_INFO.appId,
+            localCacheEnabled: false,
+            modules: [new GroupChannelModule()]
+        });
+
+
+        await sendbirdChat.connect(userIdInputValue);
+        await sendbirdChat.setChannelInvitationPreference(true);
+
+        const userUpdateParams = new UserUpdateParams();
+        userUpdateParams.nickname = userNameInputValue;
+        userUpdateParams.userId = userIdInputValue;
+        await sendbirdChat.updateCurrentUserInfo(userUpdateParams);
+
+        sb = sendbirdChat;
+        updateState({ ...state, loading: true });
+        const [channels, error] = await loadChannels();
+        if (error) {
+            return onError(error);
+        }
+        console.log(sb.currentUser.userId);
```

Comment:
Log can go

Reply:
This came from basic - but I'll remove this